src/el/sayid.el

```diff
@@ -49,8 +49,9 @@
 
 (defconst sayid-version
   (eval-when-compile
-    (lm-version (or load-file-name buffer-file-name)))
-  "The current version of `clojure-mode'.")
+    (lm-version (if (boundp 'load-true-file-name)
```

Comment:
You should add some comment here explain the need for this, as it's very far from obvious. :-)